src/main/resources/locale_de_DE.properties

```diff
@@ -367,3 +367,9 @@ words.reputation=Reputation
 words.reputationOffset=Reputationsversatz
 words.role=Rolle
 words.unknown=Unbekannt
+command.repadmin.reputation.add.user.description=Der Nutzer, um Reputation hinzuzufügen.
+command.repadmin.reputation.set.user.description=Der Nutzer, der die Reputation festlegt.
+command.repadmin.reputation.remove.user.description=Der Nutzer, um Reputation zu entfernen.
+command.repadmin.reputation.add.amount.description=Der Betrag zum Hinzufügen.
+command.repadmin.reputation.set.amount.description=Der zu setzende Betrag.
+command.repadmin.reputation.remove.amount.description=Der zu entfernende Betrag.
```

Comment:
These keys are out of order

Reply:
I will order them once I merged it. this is causing too much merge conflicts x.x

Reply:
Seems fine to do that